README.md

```diff
@@ -170,6 +170,12 @@ If you are looking for the codebase of the current production version of GATK, p
    JAVA_OPTS="-Dsamjdk.use_async_io=false" ./gatk-launch <rest of command>
 ```
 
+* By default, GATK (non-spark) uses compression level 1 for writting BAM files (fastest code but least compressed files). To change the default, run GATK like this:
+
+```
+   JAVA_OPTS="-Dsamjdk.compression_level=5" ./gatk-launch <rest of command>
```

Comment:
How does `JAVA_OPTS interact with `applicationDefaultJvmArgs` in gradle? Does it supplement `applicationDefaultJvmArgs` or replace it completely?